common/src/enums/sponsorshipStatus.ts

```diff
@@ -0,0 +1,8 @@
+export enum SponsorshipStatus {
+  None = 0,
+  Pending = 1,
+  Sent = 2,
+  Active = 3,
+  PendingRevoke = 4,
+  Revoked = 5,
```

Comment:
I think these can be combined into `toDelete`, since we don't really delineate between revoked and removed for sponsorship lifetime purposes

Reply:
ofc, we could make it `toRemove` and `toRevoke` to record who caused the deletion...